github3/repos/branch.py

```diff
@@ -418,6 +528,8 @@ class ProtectionRequiredStatusChecks(models.GitHubCore):
         https://developer.github.com/v3/repos/branches/#get-branch-protection
     .. _Required Status Checks documentation:
         https://help.github.com/articles/about-required-status-checks
+    .. _Required Status Checks API documentation:
+        https://git.io/vxrJU
```

Comment:
Where are these `git.io` links coming from? Why are we using a shortener?

Reply:
I'm using a shortener because the URL were long enough to [trigger the flake8 failure](https://travis-ci.org/sigmavirus24/github3.py/jobs/360032520) and I've seen git.io shortener has been used [before](https://github.com/sigmavirus24/github3.py/blob/develop/github3/repos/branch.py#L21)

Please let me know what would be a correct way to overcome failures like that.

Reply:
URLs on their lines on their own shouldn't trigger that. In your example you have

```
See https://...
```

But if instead you did

```
See
https://...
```

That would work fine

Reply:
I didn't see any links on their own line triggering flake8 errors when I compared the diff to the lines in Flake8's output